src/main/java/frc/utils/auto/PathPlannerUtils.java

```diff
@@ -27,6 +29,13 @@
 
 public class PathPlannerUtils {
 
+	public static final RobotConfig DEFAULT_ROBOT_CONFIG = new RobotConfig(
+		74,
+		8.6,
+		new ModuleConfig(0.048, 5.24, 0.96, DCMotor.getFalcon500(2), 60, 2),
+		0.577
+	);
+
```

Comment:
Wouldn't it be very different for each robot?

Reply:
Yes, I just put syncopa as default so we could still run pp if something went wrong with the gui. But it shouldn't happen, the config should always come from the gui

Reply:
so maybe put in the config func in swerve a param for it, so people know that they need to enter it and change it for each robot

Reply:
also are u sure about the number? especially the 2 in the   DCMotor.getFalcon500(2)